docs/content/preview/reference/drivers/nodejs/postgres-pg-reference.md

```diff
@@ -48,10 +48,6 @@ To install the async utility, run the following command:
 $ npm install --save async
 ```
 
-## Fundamentals
-
-Learn how to perform common tasks required for Node.js application development using the PostgreSQL node-postgres driver.
-
 ### Connect to YugabyteDB database
 
 Node.js applications can connect to and query the YugabyteDB database using the `pg` module. The `pg.Client` is the client we can create for the application  by passing the connection string as parameter in the constructor. The `Client.connect()` can be used to connect with database with that connection string. Use `new Client(<connection-string>)` to create a new client with the connection string.
```

Comment:
This describes a bunch of code that isn't actually on this page - it's all on the drivers and orms page

Reply:
That is how the reference pages are structured from the beginning. Maybe during a new iteration we can work on modifying it better.